packages/browser/src/scenes/library/tabs/files/LibraryExplorerScene.tsx

```diff
@@ -1,15 +1,19 @@
+import { useQuery, useSDK } from '@stump/client'
+
 import { FileExplorer } from '@/components/explorer'
 
 import { useLibraryContext } from '../../context'
 
 export default function LibraryExplorerScene() {
-	const {
-		library: { path },
-	} = useLibraryContext()
+	const { library } = useLibraryContext()
+	const { sdk } = useSDK()
+	const { data: uploadConfig } = useQuery([sdk.upload.keys.config], () => sdk.upload.config(), {
+		suspense: true,
+	})
```

Comment:
I wrote this, it will error for those without upload permission 😛 I'll try to push a fix today, it is just a matter of doing something like:

```
enabled: checkPermission('file:upload`)
```